zk-token-sdk/src/encryption/auth_encryption.rs

```diff
@@ -42,11 +43,16 @@ pub enum AuthenticatedEncryptionError {
 
 struct AuthenticatedEncryption;
 impl AuthenticatedEncryption {
+    /// Generates an authenticated encryption key.
+    ///
+    /// This function is randomized. It internally samples a 128-bit key using `OsRng`.
     #[cfg(not(target_os = "solana"))]
     fn keygen() -> AeKey {
         AeKey(OsRng.gen::<[u8; 16]>())
     }
 
+    /// On input an authenticated encryption key and an amount, the function returns a
```

Comment:
```suggestion
    /// On input of an authenticated encryption key and an amount, the function returns a
```

Reply:
Terribly sorry... should have proof read everything more carefully before creating these type of PRs.